services/app-api/forms/qm.ts

```diff
@@ -231,3 +232,9 @@ export const qmReportTemplate: ReportTemplate = {
     },
   },
 };
+
+export const praStatement: ParagraphTemplate = {
```

Comment:
There might be a few assumptions being made here. By saving it alongside our qm template we are implying it needs to be maintained at the same frequency, and uniquely for a report.

Rather my guess is this disclosure might be safe to maintain in a single place for the entire application, and may update for every report that exists, rather than a single report. Can you clarify this with Cam and Nat?

Reply:
Confirmed with Cam that for now we can assume that we might re-use it for other reports. I moved the content to the existing `constants.ts` file in `app-api/utils`